README.md

```diff
@@ -1 +1,37 @@
-# project_survey_form by Andrew and Akhtar
+# milestone-project/ phase-one 
+
+this is a project to learn how to review code and check for error
+
+### added html and css file to Github
+### created a linter to check for error in the HTML and CSS codes
+
+### created a file that should be ignored by Github
+
+### Additional description about the project and its features.
+
+## Built With
+
+### HTML
+### CSS
+### GIT
+
+
+## Major languages
+
+### HTML
+### CSS
+
+## Author
+
+###👤@AK0706
+
+### 👤@alexander16108
+
+## Show your support
+### Give a ⭐️ if you like this project!
+
+## 🤝 Contributing
+### Contributions, issues, and feature requests are welcome!
+
+### Acknowledgments
+@AK0706 for sharing with me his code to review 
```

Comment:
- [ ] please give your README.md a detailed description of your project. your README.md description should be short, simple, and easy to understand with the description of the repo.